api/app/routers/morecast_v2.py

```diff
@@ -77,6 +77,7 @@ async def get_forecasts_by_date_range(start_date: date, end_date: date, request:
                                                             precip=forecast.precip,
                                                             wind_speed=forecast.wind_speed,
                                                             wind_direction=forecast.wind_direction,
+                                                            grass_curing=forecast.grass_curing,
```

Comment:
We also need to update the API schema to add the field to `MoreCastForecastOutput`

Reply:
I think that's taken care of through its inheritance

`class MoreCastForecastOutput(MoreCastForecastInput):`